code/game/machinery/vending/vendor_types/antag/antag_clothing.dm

```diff
@@ -9,14 +9,21 @@
 
 	listed_products = list()
 
-/obj/structure/machinery/cm_vending/clothing/antag/ui_interact(mob/user, ui_key = "main", var/datum/nanoui/ui = null, var/force_open = 0)
+/obj/structure/machinery/cm_vending/clothing/antag/get_listed_products(var/mob/user)
+	if(!user)
+		var/list/all_equipment = list()
+		var/list/presets = typesof(/datum/equipment_preset)
+		for(var/i in presets)
+			var/datum/equipment_preset/eq = new i
```

Comment:
shouldn't we be deleting this eq

Reply:
Err, the return is just a list of lists. I don't think we need to delete this?

Reply:
I just misread this. Good catch.